### DISCUSSION TITLE
In my README.md, format on save is inserting extra spaces after list marker

### DISCUSSION THREAD

**Discussion by tim-rohrer** [prettier/prettier#14282] [Q&A]
Original:

```
- [ ] Do something
```
However, in vscode, format on save enabled, spaces are being automatically inserted.

```
-   [ ] Do something
```

Tbh, I don't know for certain if prettier is the culprit, but I've disabled all other known formatters. So if I should be looking elsewhere, a pointer would be great. BTW, I don't believe this used to happen 🤷‍♂️ .

Thanks in advance.

**Comment by ciresnave** [comment]
I ran into the exact same issue.  With Prettier installed, it adds extra spaces after a - but with Prettier uninstalled or disabled, it doesn't.  Does anyone know if there is a Prettier setting we could change to correct this behavior for markdown files?  I've currently had to uninstall Prettier entirely to avoid this...and that sucks because I like Prettier.
